services/libs/tinybird/pipes/search_volume.pipe

```diff
@@ -0,0 +1,32 @@
+DESCRIPTION >
+    Serves the project search volume data for the `Search queries` widget
+
+NODE searchVolume_pipe
+SQL >
+    %
+    SELECT insightsProjectId, slug, , formatDateTime(dataTimestamp, '%Y-%m-%d') AS dataTimestamp, volume, updatedAt
+    FROM searchVolume FINAL
+    WHERE
+        1 = 1
+        {% if defined(startDate) %}
+            AND searchVolume.dataTimestamp
+            >= {{
+                DateTime(
+                    startDate,
+                    description="Select records with dataTimestamp only after the given date",
+                    required=False,
+                )
+            }}
+        {% end %}
+        {% if defined(endDate) %}
+            AND searchVolume.dataTimestamp
+            <= {{
+                DateTime(
+                    endDate,
+                    description="Select records with dataTimestamp only before the given date",
+                    required=False,
+                )
+            }}
+        {% end %}
+        {% if defined(insightsProjectId) %} AND searchVolume.insightsProjectId = {{ insightsProjectId }} {% end %}
```

Comment:
If you apply the filter like this here
`insightsProjectId = (select insightsProjectId from segments_filtered)`, it'll also check if the project is enabled. r. Currently, it doesn't check that, and if the project is disabled, data will be returned even though the project is disabled.

Reply:
Good one, I'll add it. Thanks! :+1: